sql/core/src/main/scala/org/apache/spark/sql/functions.scala

```diff
@@ -761,6 +761,14 @@ object functions {
    */
   def bitwiseNOT(e: Column): Column = BitwiseNot(e.expr)
 
+  /**
+   * Parses the comma separated expression strings into the columns they represent, similar to
+   * DataFrame.selectExpr
+   *
```

Comment:
it'd be great to give an example in the function here, since this is a somewhat unique function. maybe give an example with groupby?